src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -125,7 +125,7 @@ struct kds_scu_stat
       boost::char_separator<char> sep(",");
       tokenizer tokens(line, sep);
 
-      if (std::distance(tokens.begin(), tokens.end()) != 4)
+      if (std::distance(tokens.begin(), tokens.end()) != 7) 
```

Comment:
Is it possible to have this value defined in a common place.  For example, in the kds_core.h file.  This way, this DR check would never need to be updated.

Reply:
Fixed. Thanks.

Reply:
The communication between driver and userspace are through sysfs file. To generalize this we need to update this value from driver to sysfs file and userspace has to read that value from sysfs file. 
This need some more code changes  from both driver and userspace side. As this PR targeted for U30 device, I don't want to make changes at this point of time. But for master we can make that changes. 
Created a CR to track this. https://jira.xilinx.com/browse/CR-1106063

Reply:
Currently added a MACRO in the same file to avoid hard code the value.